root/report/DuplicateArtists.js

```diff
@@ -38,22 +38,26 @@ const DuplicateArtists = ({
 
       <ul>
         <li>
-          {exp.l(
-            `This report aims to identify artists with very similar names.
-             If two artists are actually the same, please merge them
-             (remember to {how_to_write_edit_notes|write an edit note}
-             and give your proof). If they\'re different, add
-             {disambiguation_comment|disambiguation comments} to them
-             (and once a group of similarly named artists have
-             disambiguation comments, they will stop appearing here).`,
-            {
-              disambiguation_comment: '/doc/Disambiguation_Comment',
-              how_to_write_edit_notes: '/doc/How_to_Write_Edit_Notes',
-            },
-          )}
+          {exp.l(`This report aims to identify artists with very similar
+                  names. If two artists are actually the same, please merge
+                  them (remember to {how_to_write_edit_notes|write an edit
+                  note} and give your proof). If they\'re different, add
+                  {disambiguation_comment|disambiguation comments} to them
+                  (and once a group of similarly named artists have
+                  disambiguation comments, they will stop appearing here).`,
+                 {
+                   disambiguation_comment: '/doc/Disambiguation_Comment',
+                   how_to_write_edit_notes: '/doc/How_to_Write_Edit_Notes',
+                 })}
```

Comment:
There are a few changes like this, but I'm not sure I'd call them improvements. I think both of these styles are good depending on the circumstances, e.g. the changes to the two strings below look good. But here, we'd just be adding 5 extra spaces of indentation, giving these very long lines even less space. The style on the left works well for avoiding too much indentation when you have really long arguments to fit.

I also like how the one on the left keeps substitutions like `{how_to_write_edit_notes|write an edit note}` on a single line. It's a really small thing, but makes it easier to read the text and parse the syntax.

Reply:
@yvanzo: do you want to revisit this (since it's your commit) or should I? :)

Reply:
@reosarevok: I just pushed three more commits to address this, normalizing indentation the suggested way instead, should probably be squashed within https://github.com/metabrainz/musicbrainz-server/pull/934/commits/ae2c3a78c0475ff75bfd2633ecf7b7df715967c1 once accepted.

I agree with _avoiding too much indentation when you have really long arguments to fit_. Should starting multi-line message on a new line be added to MBS coding style?

@mwiencek: Does breaking line in the middle of substitution defeat the current parser? (I don’t really find it any different to read.) Could that be fixed in the parser or should messages be automatically tested?

Reply:
It doesn't break the parser if the line breaks come after the `{var|` part, no. As for whether it's easier to read, I guess it's just me, so never mind. :)

Reply:
I've generally tried to keep those in one line, although sometimes they're so long they won't even fit anyway, and then I've just split them because it's not that important IMO.

Reply:
Ok, no worry, I did not break any line within substitution here anyway.